cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java

```diff
@@ -63,23 +64,36 @@ public Network importNetwork(String cgmUrl) {
         return Network.read(FileUtil.getFilenameFromUrl(cgmUrl), openUrlStream(cgmUrl));
     }
 
-    public CseCrac importCseCrac(String cracUrl) {
+    public CRACDocumentType importCseCrac(String cracUrl) {
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+
+        CRACDocumentType cracDocumentType;
```

Comment:
Duplicate from export_runner's `FileImporter`.

Reply:
As discussed, this point should be handled in a separate ticket dedicated to refactoring. Won't be changed in this PR.